plugins/woocommerce/includes/admin/helper/class-wc-helper-admin.php

```diff
@@ -60,6 +62,7 @@ function ( $product ) {
 			'wooUpdateManagerActive'     => WC_Woo_Update_Manager_Plugin::is_plugin_active(),
 			'wooUpdateManagerInstallUrl' => WC_Woo_Update_Manager_Plugin::generate_install_url(),
 			'wooUpdateManagerPluginSlug' => WC_Woo_Update_Manager_Plugin::WOO_UPDATE_MANAGER_SLUG,
+			'wooUpdateCount'             => WC_Woo_Update_Manager_Plugin::increment_update_count_for_woo_update_manager( WC_Helper_Updater::get_updates_count() ?? 0 ),
```

Comment:
The logic of getting the count here duplicates the logic in `get_updates_count_html`. Would it be possible to standardize and have just one function that returns the final count?

Reply:
Fixed here: https://github.com/woocommerce/woocommerce/pull/46088/commits/f09891b9b7f1beca9088881dae7d969b2bf50bc9